sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestGetGeneratedKeysPostgres.java

```diff
@@ -83,4 +91,103 @@ public void testBatch() throws Exception {
             assertThat(dao.findNameById(ids[1])).isEqualTo("Macklin");
         });
     }
+
+    @Test
+    public void testUseRowMapperUpdate() throws Exception {
+        dbRule.getJdbi().useHandle(handle -> {
+            handle.execute("drop table something");
+            handle.execute("create table something (id serial, name varchar(50), created_on timestamp default now())");
+
+            UseRowMapperDao dao = handle.attach(UseRowMapperDao.class);
+
+            IdCreateTime result = dao.insert("foo");
+            assertThat(result.id).isEqualTo(1);
+            assertThat(result.createdOn).isNotNull();
+        });
+    }
+
+    @Test
+    public void testUseRowMapperBatch() throws Exception {
+        dbRule.getJdbi().useHandle(handle -> {
+            handle.execute("drop table something");
+            handle.execute("create table something (id serial, name varchar(50), created_on timestamp default now())");
+
+            UseRowMapperDao dao = handle.attach(UseRowMapperDao.class);
+
+            List<IdCreateTime> results = dao.insertBatch("foo", "bar");
+            assertThat(results).extracting(ic -> ic.id).containsExactly(1L, 2L);
+            assertThat(results).extracting(ic -> ic.createdOn).hasSize(2).doesNotContainNull();
+        });
+    }
+
+    public interface UseRowMapperDao {
```

Comment:
Just a nit, but I think it would be better to move the `UserRowMapperDao` class after the tests to `RegisterRowMapperDao`.  I think it's better from the point of maintaining a consistent order of the items in the class.